tests/integ/sagemaker/lineage/test_artifact.py

```diff
@@ -131,9 +132,14 @@ def validate():
 
 
 def test_downstream_trials_v2(trial_associated_artifact, trial_obj, sagemaker_session):
-    trials = trial_associated_artifact.downstream_trials_v2()
-    assert len(trials) == 1
-    assert trial_obj.trial_name in trials
+    for _ in retries(
```

Comment:
Just a FYI: if need a backoff retry in the future, here's the helper function: https://github.com/aws/sagemaker-python-sdk/blob/c02d74b83cbb42800d7962caabf4e828b8603f7c/src/sagemaker/utils.py#L607